### PR TITLE
Add RockyLinux8 images

### PR DESCRIPTION
**Pull request by dkasabovn**
### Changes
- Added Rockylinux8 Docker images
- Modified GOSS test files where users/groups where being checked by hardcoded gid/uid due to Centos-based os' allocation of users/groups
- Renamed install-r.sh script to install-r-bionic.sh to increase clarity for anyone working with `workbench-for-microsoft-azure-ml` in the future.
- For r-session-complete `centos7` image I added a flag to ensure that we fail builds if we try to pull non-existent packages

### CODE REVIEW

**Review by melissa-barca — APPROVED** [review]
LGTM!

File: connect/Dockerfile.rockylinux8 [hunk 1]
```diff
@@ -0,0 +1,128 @@
+ARG R_VERSION=3.6.2
+FROM rstudio/r-base:${R_VERSION}-rockylinux8
+LABEL maintainer="RStudio Docker <docker@rstudio.com>"
+
+# Locale configuration --------------------------------------------------------#
+
+# hadolint ignore=DL3008,DL3009
```
Comment (ianpittwood) [inline connect/Dockerfile.rockylinux8:7]: Let's remove these in favor of using the `hadolint.yaml` file.

**Comment by ianpittwood** [comment]
Temporarily closing to test out how RL8 will affect our security scanning.

**Comment by ianpittwood** [comment]
We're still contemplating if we need a RHEL 8 image as well as which one we'll use. Closing this PR for the time being.

**Pull request closed without merging**